model-server-openapi/specifications/model-server-operative.yaml

```diff
@@ -34,3 +43,11 @@ components:
         text/plain:
           schema:
             type: string
+  schemas:
+    AboutInformation:
+      type: object
+      properties:
+        version:
+          type: string
```

Comment:
We could further specify the format with a regex.

Reply:
We could do it. 
There is even a recommended Regex for [SemVer](https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string). 
But currently I see no benefit in specifying the pattern of the version.

I would rather not specify it yet.
This would entail doing additional checks in the Gradle build.
And maybe under some circumstances, we will want a version that does not match it.

Reply:
@languitar I will merge the commit as is with the type "string".
I will come back to you to talk about, whether it makes sens to make it stricter with an Regex.